independent-publisher-connectors/GoQR/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",

```

Comment:
@rsantos00  Can you change it default iconBrandColor for all the IP connectors?

"iconBrandColor": "#da3b01"

Reply:
fixed